src/lib/courseChannels.js

```diff
@@ -137,4 +142,66 @@ export async function unlinkExistingCourseChannel(roleId, guild) {
     }
 
     await course.destroy();
+}
+
+/**
+ * Updates the colors of the course roles in the server
+ */
+export function updateCourseColors(guild) {
```

Comment:
Since you use JSDocs in the latter function, you should also include types for `guild` here.